### ISSUE TITLE
I created a web scraper for adjusting the star count of the plugins

### ISSUE THREAD

**Issue by MemerGamer** [darksinge/plink.nvim#1]
### Description
I wanted to share a mini project I've recently developed called Plink Star Scraper. It's a simple web scraper that retrieves the star count of Neovim plugins and saves it to a `.json` file.

### Project Link
[Link to Plink Star Scraper](https://github.com/MemerGamer/plink-star-scraper)

### Limitations
1. It might not be the best way to get the data we want, but it's working. I'm open to suggestions on how to improve the scraper.

2. To avoid overloading the API, I focused my testing on the keyword "telescope."

### How to Use
I have included detailed instructions in the project's README file on how to run the script and retrieve star counts for Plink plugins. I have also included a sample input and output file in the project's root directory.

**Comment by darksinge** [comment]
First of all, I think it's awesome there's enough interest in this plugin to have someone open an issue 😆. Fetching github stars is on the roadmap and I do appreciate the help!

Ideally, the code for fetching and updating the stargazer count would be in this project and written in Javascript/Typescript/Bash (read: easily executable script), and probably run with github actions as a scheduled worker.

I was also planning to use github's API to fetch this data, as the stargazer count is readily available. For example:

```
curl 'https://api.github.com/repos/darksinge/plink.nvim' -H 'Accept: application/json' -H "Authorization: Bearer $GITHUB_TOKEN" | jq .
{
  ...,
  "stargazers_count": 8,
  ...,
}
```

I genuinely appreciate the interest. If you want to open a PR I'd be happy to take a look!

**EDIT**: This functionality actually belongs to the backend project [github.com/darksinge/plink-nvim-site](https://github.com/darksinge/plink-nvim-site), so PRs would be welcome over there.

**Comment by MemerGamer** [comment]
Alright, I'll close this issue now and take a look at the backend repository. I think I can do a better job with the scraping using TypeScript since I'm more experienced with that language. Initially, I just wanted to try out the idea I got after watching your video, and since I'm currently learning Go, I figured why not give it a shot. Once I have something functional, I'll create a pull request in the backend repository. And of course, I'll make use of the GitHub API for that. 

Thank you for your response!
